frontend/packages/console-shared/src/components/alerts/SwitchToYAMLAlert.tsx

```diff
@@ -12,6 +12,7 @@ const SwitchToYAMLAlert: React.FC<{ onClose?: () => void }> = ({ onClose }) => {
         'console-shared~Note: Some fields may not be represented in this form view. Please select "YAML view" for full control.',
       )}
       variant="info"
+      data-testid="info-alert"
```

Comment:
I think we are going forward with using `data-test` in cypress tests. Using a consistent attribute would be better. `screen.ByTestId()` supports `data-testid` by default but we can configure it to use `data-test` in our case using configure option https://testing-library.com/docs/queries/bytestid/#overriding-data-testid